packages/sdk-size-test/app/routes/$.tsx

```diff
@@ -0,0 +1,25 @@
+import type {
+  LinksFunction,
+  LoaderArgs,
+  MetaFunction,
+} from "@remix-run/cloudflare";
+import { useLoaderData } from "@remix-run/react";
+import { InstanceRoot, Root } from "@webstudio-is/react-sdk";
+import { getComponent } from "../page-components/page-main";
+
+export const meta: MetaFunction = () => {
+  return { title: "test" };
+};
+
+export const links: LinksFunction = () => [];
+
+export const loader = async ({ context, request }: LoaderArgs) => {
```

Comment:
no need, for that either

Reply:
Just tried be as much as near to existing saas app. Also have in mind that probably it will be example app some day

Reply:
we will, probably, but its unlikely going to be this test suit, we need something super easily maintainable for such thing, ideally the real thing we use to bundle, but if not something super minimal